src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/import_features_random_conditions.json

```diff
@@ -0,0 +1,28 @@
+{
+  "Color": "Red",
+  "Region": "West US",
+  "FeatureManagement": {
+    "Beta": false,
+    "Percentage": true,
+    "Timestamp": {
+      "EnabledFor": [
+        {
+          "Name": "Local Tests",
+          "Parameters": {
+            "StartTime": "2019-01-01T00:00:00Z",
+            "EndTime": "2019-09-01T00:00:00Z"
+          }
+        },
+        {
+          "Name": "Production Tests",
+          "Parameters": {
+            "StartTime": "2019-09-02T00:00:00Z",
+            "EndTime": "2019-11-01T00:00:00Z"
+          }
+        }
+      ],
+      "requirement_type": "All",
```

Comment:
RequirementType should follow the same naming convention as EnabledFor and FeatureManagement. 
.NET FeatureManagement library expects PascalCase: https://github.com/albertofori/azure-cli/blob/36786c51a5d42ff1b0a197e72a2c12261ff6743b/src/azure-cli/azure/cli/command_modules/appconfig/_params.py#L189

Reply:
This has been updated, thanks!

Reply:
I think we need to update the code to support different naming conventions for requirement type.

Reply:
Thanks for pointing this out. It has been updated accordingly.